### DISCUSSION TITLE
Curious Trajectories in IntraCellular Mucosa

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1246] [Questions & Problems] [answered]
Using the model provided in the Git library for Itraconazole, I plotted trajectories for drug amount in various mucosa intracellular compartments of the lower intestine. The trajectories seem rather strange, and I am trying to understand what's going on. Could anyone explain how such a precipitous drop occurs? 

Ive attached a photo with plots of the lower intestine mucosa colon sigmoid, and the lower intestine mucosa colon descendens, but I believe all of the lower intestine mucosa intracellular compartments have similar trajectories. The corresponding plasma trajectories are also strange, I've attached a photo with those as well.

Thanks,

DS

<img width="615" alt="Screen Shot 2022-07-08 at 4 23 47 PM" src="https://user-images.githubusercontent.com/25123758/178065327-10c22ee7-a479-4fd5-a354-a0b79ea02254.png">
<img width="604" alt="Screen Shot 2022-07-08 at 4 24 23 PM" src="https://user-images.githubusercontent.com/25123758/178065331-f15c110d-786b-4f7b-bca3-6dd49994b20c.png">

**Comment by Yuri05** [comment, marked as answer]
This is the result of **solubility limitation**.
If you look at the plot below you will see that the **passive absorption rate** `Lumen|ColonDescendes ==> Mucosa|ColonDescendes|Intracellular` (dashed curve) rapidly decreases at ca. t=81h - which leads to the rapid decrease of the intracellular concentration in `Mucosa|ColonDescendes|Intracellular` at the same time point.

Passive absorption rate is proportional to the "_Effective concentration in Lumen_" - and if you look at it in the plot (dotted curve) - you will see again the rapid decrease - which is the reason for the change of passive absorption rate.
![grafik](https://user-images.githubusercontent.com/25061876/178246074-bd9a1667-ab80-4783-914e-403f51f18715.png)

"_Effective concentration in Lumen_" in case of "no supersaturation" (which is the default setting - also for Itraconazole) is defined as 
`Min (Drug amount in lumen segment / Liquid volume in lumen segment; Solubility in lumen segment)` - **this is the solubility limitation**.

Now if you plot the `Effective concentration` vs. `Drug amount in lumen segment / Liquid volume in lumen segment`: it looks like this

![grafik](https://user-images.githubusercontent.com/25061876/178248853-5210a9e5-ea21-43eb-ad8e-662128ec26a7.png)

Here you can perfectly see how the solubility cut off leads to this bend in the effective concentration curve, finally leading to what you observed.

And if you disable the solubility limitation (by setting `Enable supersaturation = YES` for Itraconazole) - you will see smooth curves without this effect:
(CAUTION: for the formulations in the Itraconazole project it does not make any sense - it is just to demonstrate the effect)
![grafik](https://user-images.githubusercontent.com/25061876/178249696-2376f247-9e37-4999-a9ad-a7c780dace0c.png)

**Reply by DonkeySaddle** [reply]
Thank you for the thorough explanation. Really appreciate it, this is great!

DS
